### PR TITLE
Add cute little script for quick linux configuration

### LINKED ISSUE #238: Set Up API to Work On New  Server
**Is this related to a problem? Please describe.**
The server is set up but our current API branch is only configured for local access? 

**Describe the solution you'd like**
Mess with the port settings, and figure out how to properly send and receive requests. Use Postman and Jmeter to verify

**Requirements for delivery**
A new branch with updated API setting son it

### PR DESCRIPTION
**Pull request by mkuelker**
This is just a little something I wrote to pad my dismal gitstats for the week. 

A lot of my time went into research and configuring ec2 in AWS, but I figured I'd also leave this here.

### CODE REVIEW

File: docs/research/API_load_test/UbuntuSetup.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sudo apt install -y nodejs
+node --version
+echo "Installing ffmpeg"
+sudo apt update && sudo apt upgrade
+sudo apt install ffmpeg
+ffmpeg -version
+echo "Installing Rhubarb"
+sudo apt update
+sudo apt-get install unzip
+curl -LO https://github.com/DanielSWolf/rhubarb-lip-sync/releases/download/v1.13.0/Rhubarb-Lip-Sync-1.13.0-Linux.zip
```
Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh:28]: this is a cool way of installing rhubarb - nicely done!

File: docs/research/API_load_test/UbuntuSetup.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+unzip Rhubarb-Lip-Sync-1.13.0-Linux.zip
+mv Rhubarb-Lip-Sync-1.13.0-Linux rhubarb
+rm Rhubarb-Lip-Sync-1.13.0-Linux.zip
+echo "Install app dependencies"
+npm install --save-dev
```
Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh]: we actually probably dont want to be installing dev dependencies within the production environment. 

At the current moment, I don't know that we have any production dependencies so the NPM script should probably be updated!

Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh]:
```suggestion
npm install --production
```
try this out - we should only need express & multer installed to run the production environment!

Comment (mkuelker) [reply docs/research/API_load_test/UbuntuSetup.sh]: Ooh! will do!

File: docs/research/API_load_test/UbuntuSetup.sh [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ffmpeg -version
+echo "Installing Rhubarb"
+sudo apt update
+sudo apt-get install unzip
+curl -LO https://github.com/DanielSWolf/rhubarb-lip-sync/releases/download/v1.13.0/Rhubarb-Lip-Sync-1.13.0-Linux.zip
+unzip Rhubarb-Lip-Sync-1.13.0-Linux.zip
+mv Rhubarb-Lip-Sync-1.13.0-Linux rhubarb
+rm Rhubarb-Lip-Sync-1.13.0-Linux.zip
+echo "Install app dependencies"
+npm install --production
```
Comment (LoganParker) [inline docs/research/API_load_test/UbuntuSetup.sh:33]: Let me know if this change worked! Just keeps our footprint small

**Review by LoganParker — APPROVED** [review]
Awesome work matt :) thanks for setting all this up - server setup always boggles my mind so im glad you were able to get it working!

**Review by vnajack — APPROVED** [review]

**Pull request merged**